circuits/cpp/src/aztec3/utils/types/convert.hpp

```diff
@@ -130,6 +131,28 @@ template <typename Composer, std::size_t SIZE> std::array<std::optional<typename
     return map(arr, ref_to_ct);
 };
 
+/**
+ * @brief Convert from an array of any native types (NT_TYPE) to array of circuit types (CT_TYPE)
+ */
+template <typename Composer, typename CT_TYPE, typename NT_TYPE, std::size_t SIZE>
+std::array<CT_TYPE, SIZE> to_ct(Composer& composer, std::array<NT_TYPE, SIZE> const& arr)
+{
+    auto ref_to_ct = [&](NT_TYPE const& e) { return e.to_circuit_type(composer); };
+
+    return map(arr, ref_to_ct);
+};
+
+/**
+ * @brief Convert from an array of any native types (NT_TYPE) to array of circuit types (CT_TYPE)
```

Comment:
Comment might mention "optional" otherwise it is the very same comment as the first routine.